### DISCUSSION TITLE
Research topic related to Koordinator

### DISCUSSION THREAD

**Discussion by xhejtman** [koordinator-sh/koordinator#1608] [Ideas]
Hello,

we are discovered this project while doing research on the very same topic - resource management in containers/Kubernetes. The Koordinator already implements many of ideas we had as well. 

I wonder, if somebody here around would have an idea to contribute to Koordinator in more academic/research way that we could do. Any idea is appreciated, but basically, the idea has to be suitable for paper publication, not purely implementation of something known. Thanks a lot!

**Comment by zwzhang0107** [comment]
Hi @xhejtman, thank you for following us.
We do have some ongoing research topic about QoS management, which aims to detect and solve node resource competition when running different kinds of workloads on K8s
Besides, Koordinator only improves QoS for CPU workloads, and QoS of GPU workloads is worthy of study.
Any discussion is welcome if you have more ideas.
